internal/cmd/schema-registry/command.go

```diff
@@ -46,9 +46,20 @@ func New(cfg *v1.Config, prerunner pcmd.PreRunner, srClient *srsdk.APIClient) *c
 func addCompatibilityFlag(cmd *cobra.Command) {
 	compatibilities := []string{"backward", "backward_transitive", "forward", "forward_transitive", "full", "full_transitive", "none"}
 	cmd.Flags().String("compatibility", "", fmt.Sprintf("Can be %s.", utils.ArrayToCommaDelimitedString(compatibilities, "or")))
+	cmd.Flags().String("compatibility-group", "", "The name of the compatibility group.")
+	cmd.Flags().String("metadata-defaults", "", "The path to the schema metadata defaults file.")
+	cmd.Flags().String("metadata-overrides", "", "The path to the schema metadata overrides file.")
+	cmd.Flags().String("ruleset-defaults", "", "The path to the schema ruleset defaults file.")
+	cmd.Flags().String("ruleset-overrides", "", "The path to the schema ruleset overrides file.")
```

Comment:
Can you write some examples for these flags too? Are they typically supplied together? Or in certain combinations?

Reply:
There are examples in `command_cluster_update.go`, `command_subject_update.go` and etc. There's no typical use case, user are free to set default or override depending on their needs.